rust/extractor/src/translate/base.rs

```diff
@@ -20,62 +23,30 @@ use ra_ap_syntax::{
 #[macro_export]
 macro_rules! emit_detached {
     (MacroCall, $self:ident, $node:ident, $label:ident) => {
-        $self.extract_macro_call_expanded(&$node, $label.into());
+        $self.extract_macro_call_expanded(&$node, $label);
```

Comment:
Out of curiosity, if the `into()` here wasn't needed, why is it needed for all the other lines below?

Reply:
That function is only called on macros and macro labels. The other calls below accept an `Item` label, which is a super-type of the specific label being produced in the extractor and therefore requires the conversion.